src/hevm/src/EVM.hs

```diff
@@ -1625,6 +1625,13 @@ cheatActions =
     [ action "warp(uint256)" [AbiUIntType 256] $
         \[AbiUInt 256 x] -> do
           assign (block . timestamp) (w256 (W256 x))
+          return Nothing,
+      action "store(address,bytes32,bytes32)" [AbiAddressType, AbiBytesType 32, AbiBytesType 32] $
+        \[AbiAddress a, AbiBytes 32 x, AbiBytes 32 y] -> do
+          let slot = w256 $ word x
+              new = w256 $ word y
+          assign (env . contracts . ix a . storage . at slot)
```

Comment:
Will this write to the slot if `a` doesnt exist in the `contracts` already? Im not sure. You may wanna fetch the contract first

Reply:
This would make it play nicer with rpc mode

Reply:
Great point, currently this ends up getting overwritten if the contract is fetched later.

Reply:
Fixed. I tested it out and indeed now the `hevm.store()` cheatcode commutes with RPC fetch, as desired.